sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorExporterEventSource.cs

```diff
@@ -71,5 +74,38 @@ private static string GetMessage(object value)
         {
             return value is Exception exception ? exception.ToInvariantString() : value.ToString();
         }
+
+        public class AzureMonitorExporterEventListener : EventListener
+        {
+            private readonly List<EventSource> eventSources = new List<EventSource>();
+
+            public override void Dispose()
+            {
+                foreach (EventSource eventSource in this.eventSources)
+                {
+                    this.DisableEvents(eventSource);
+                }
+
+                base.Dispose();
+                GC.SuppressFinalize(this);
+            }
+
+            protected override void OnEventSourceCreated(EventSource eventSource)
+            {
+                if (eventSource?.Name.StartsWith("Microsoft-OpenTelemetry", StringComparison.OrdinalIgnoreCase) == true)
```

Comment:
nit: the EventSourceName is a constant (line 14).
Would it make sense to match that value here instead of using StartsWith?